src/containers/shared/components/Sequence.tsx

```diff
@@ -31,19 +31,17 @@ const Sequence = (props: Props) => {
 };
 
 Sequence.propTypes = {
-  t: PropTypes.func,
   sequence: PropTypes.number,
   ticketSequence: PropTypes.number,
   addContextHelp: PropTypes.bool,
   account: PropTypes.string,
 };
 
 Sequence.defaultProps = {
-  t: (d: string) => d,
   sequence: PropTypes.number,
   ticketSequence: 0,
   addContextHelp: false,
   account: '',
 };
 
-export default translate()(Sequence);
+export default Sequence;
```

Comment:
Why is Sequence no longer being translated?

Reply:
`translate(...)` populated the `t` prop.  Now `useTranslation()` is used which makes t available via a hook.